ДЗ 21 спринта

ZarubaGit opened the pull request:
1. Внедрение библиотеки Room в проект
2. Введение функционала по добавлению трека в избранно  с помощью базы данных

ZarubaGit committed:
ДЗ 21 спринта
1. Внедрение библиотеки Room в проект
2. Введение функционала по добавлению трека в избранно  с помощью базы данных

ZarubaGit committed:
Изменил цвет кнопки в ночном режиме

Grigoriy-M commented on app/src/main/java/com/example/playlistmaker/ui/media/FavoriteSongsFragment.kt:
```diff
@@ -21,7 +34,63 @@ class FavoriteSongsFragment : Fragment() {
+    }
+
+    private fun showEmpty() {
+        binding.media.visibility = View.VISIBLE
```
![view](https://github.com/ZarubaGit/PlaylistMaker/assets/151455241/769fc1d9-8808-406a-8055-15c26c90dc5b)

ZarubaGit committed:
1. Добавли ссылку на приложение
2. Теперь трэк не сбрасывает состояние кнопки "Избранное"
3. Проверка состояния на пустое поле ввода и отображение иконки, которая удаляет содержимое
